lib/determine-basal/determine-basal.js

```diff
@@ -695,7 +695,7 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
         }
 
         // set eventualBG based on COB or UAM predBGs
-        rT.eventualBG = eventualBG;
+        rT.eventualBG = convert_bg(eventualBG,profile);  // for FAX needs to be in mg/dL
```

Comment:
What is FAX? Are you intending to comment out this line in light of #1439? Looks like it's still uncommented.

Reply:
If you want to comment out / revert this one for now until we figure out #1439, the other console.error lines are perfectly safe to change, and I can merge this PR with just those.

Reply:
yes FAX = FreeAPS-X, there the rT variables are used to populate the swift code

Reply:
do you have an idea how to convert the prediction values UAM, ZT and the like as well?

Reply:
> if you want to comment out / revert this one for now

ok, forgot about it - yes, I took it out now